src/aap_eda/core/models/user.py

```diff
@@ -31,6 +31,7 @@ class User(AbstractUser):
 
     roles = models.ManyToManyField("Role", related_name="users")
     modified_at = models.DateTimeField(auto_now=True, null=False)
+    is_service_account = models.BooleanField(default=False)
```

Comment:
We already have a boolean `is_superuser`, what's the difference between it with this new `is_service_account`? Can we use the existing `is_superuser`?

Reply:
For the superuser, its `is_service_account` will also have the default `false` value, right? how it can be excluded?

Reply:
The filter is `(is_service_account=False)`, so superuser and other regular users will be included in the list.